src/cache_refresh/cache_refresh.cc

```diff
@@ -48,6 +49,11 @@ static const uint64_t kNssPasswdCacheSize = 2048;
 // exceed 32k.
 static const uint64_t kPasswdBufferSize = 32768;
 
+bool fexists(const char *filename) {
```

Comment:
no vanity functions, please. you can call stat in code where this was called.

Reply:
Done.